ui/src/components/table/CustomTableRow.jsx

```diff
@@ -73,6 +74,21 @@ function CustomTableRow(props) {
                             />
                         </Tooltip>
                     )}
+                    {rowActions.includes('search') && (
+                        <Tooltip content={_('Search')}>
+                            <ActionButtonComponent
+                                appearance="flat"
+                                icon={<Magnifier screenReaderText={null} size={1} />}
+                                to={`/app/search/search?q=search%20index%3D_internal%20source%3D*${selectedRow.name}*`}
+                                className="searchBtn"
+                                style={{
```

Comment:
is there a way to include in ActionButtonComponent?

Reply:
Done